PullToRefresh/PullToRefresh.swift

```diff
@@ -199,20 +199,30 @@ class DefaultRefreshView: UIView {
     override func layoutSubviews() {
         if (activicyIndicator == nil) {
             activicyIndicator = UIActivityIndicatorView(activityIndicatorStyle: UIActivityIndicatorViewStyle.Gray)
-            activicyIndicator.center = convertPoint(center, fromView: superview)
             activicyIndicator.hidesWhenStopped = false
             addSubview(activicyIndicator)
         }
-        
+        centerActivityIndicator()
+        frameInSuperview(superview)
         super.layoutSubviews()
     }
     
     override func willMoveToSuperview(newSuperview: UIView?) {
         super.willMoveToSuperview(newSuperview)
+        frameInSuperview(superview)
+    }
+    
+    private func frameInSuperview(newSuperview: UIView?) {
```

Comment:
@EgorSobko could you plz rename this method to something like setupFrameInSuperview or similar ? It makes it more readable. Thanks.